portal-web/test/functional/com/liferay/portalweb/macros/Clustering.macro

```diff
@@ -235,8 +227,7 @@ definition {
 		Clustering._viewNoLiveSession(count = ${countNotPresent});
 	}
 
-	@description = "This will view the text not shows at a specific bundle at ${base_branch}-bundles-${bundle_index}/logs."
-	@summary = "Default summary"
+	@summary = "This will view the text not shows at a specific bundle at ${base_branch}-bundles-${bundle_index}/logs."
 	macro viewNoSessionReplicateLogErrorOnSpecificNode(exceptionText = null, nodePort = null) {
```

Comment:
This seems the same as the one on the bottom. This macro looks like it might have been intended to assert a specific message but it's the same. I know this PR is just for the summaries though but we should combine these eventually. Right now this should also be

"This will assert ${exceptionText} is no present in the log of node ${nodePort}"

Reply:
Btw I reviewed these in a different order so bottom is actually top I guess